x-pack/plugins/spaces/index.js

```diff
@@ -42,10 +42,11 @@ export const spaces = (kibana) => new kibana.Plugin({
     hacks: [],
     mappings,
     home: ['plugins/spaces/register_feature'],
-    injectDefaultVars: function () {
+    injectDefaultVars: function (server) {
       return {
         spaces: [],
-        activeSpace: null
+        activeSpace: null,
+        rootBasePath: server.config().get('server.basePath'),
```

Comment:
How would you feel about renaming this to "spaceSelectorURL" or something like that?

Reply:
Good idea -- it better reflects the intent/usage of this variable